guides/common/modules/con_host-substatus-overview.adoc

```diff
@@ -52,7 +52,7 @@ When your host uses a configuration management system but {Project} does not rec
 Otherwise it is mapped to OK.
 
 .Search syntax
-If you want to search for hosts according to their sub-status, use the syntax for searching in {Project} that is outlined in the {AdministeringDocURL}Searching_and_Bookmarking_admin[] chapter of the _Administering {Project}_ guide, and then build your searches out using the following status-related examples:
+If you want to search for hosts according to their sub-status, use the syntax for searching in {Project} that is outlined in the {AdministeringDocURL}Searching_and_Bookmarking_admin[Searching and Bookmarking] chapter of the _Administering {Project}_ guide, and then build your searches out using the following status-related examples:
```

Comment:
Could we make this one into 3 sentences?
```
If you want to search for hosts according to their sub-status, use the syntax for searching in {Project}.
For more information, see {AdministeringDocURL}Searching_and_Bookmarking_admin[Searching and Bookmarking] in the _Administering {Project}_ guide.
Then build your searches out using the following status-related examples:
```

Reply:
Applied.